scripts/eks-setup.sh

```diff
@@ -16,7 +16,7 @@ sed -i -e "s/__AOS_ENDPOINT__/$AOS_ENDPOINT/g" observability-with-amazon-opensea
 sed -i -e "s/__AOS_USERNAME__/$AOS_USERNAME/g" observability-with-amazon-opensearch/sample-apps/01-data-preper/kubernetes/data-preper.yaml
 sed -i -e "s/__AOS_PASSWORD__/$AOS_PASSWORD/g" observability-with-amazon-opensearch/sample-apps/01-data-preper/kubernetes/data-preper.yaml
```

Comment:
How are you retrieving these credentials via script?

Reply:
The AOS credentials are in Cloudformation output like before.
this command in eks-setup.sh gets the latest AOSPassword - 
`export AOS_PASSWORD=$(aws cloudformation describe-stacks --stack-name ${STACK_NAME} --query "Stacks[0].Outputs[?OutputKey=='AOSDomainPassword'].OutputValue" --output text)`